src/components/v2/values/ea/EAValues.component.tsx

```diff
@@ -89,10 +95,9 @@ function ExternalAppValues({ appId }: { appId: string }) {
 
                     setChartValues(_chartValues)
                     setChartValuesList([_chartValues])
+                    setModifiedValuesYaml(YAML.stringify(JSON.parse(_releaseInfo.mergedValues)))
+                    setIsLoading(false)
```

Comment:
Don't we need setIsLoading(false) for if case?

Reply:
It's being handled & set in initData function.